dot/sync/message.go

```diff
@@ -84,35 +84,35 @@ func (s *Service) CreateBlockResponse(blockRequest *network.BlockRequestMessage)
 		blockData.Justification = optional.NewBytes(false, nil)
 
 		// header
-		if (blockRequest.RequestedData & 1) == 1 {
+		if (blockRequest.RequestedData & network.RequestedDataHeader) == 1 {
 			retData, err := s.blockState.GetHeader(hash)
 			if err == nil && retData != nil {
 				blockData.Header = retData.AsOptional()
 			}
 		}
 		// body
-		if (blockRequest.RequestedData&2)>>1 == 1 {
+		if (blockRequest.RequestedData&network.RequestedDataBody)>>1 == 1 {
```

Comment:
could you add spaces around thhe `&` for consistency?

Reply:
for some reason gofmt wants it this way, when I make lint with the spaces I get:
```
dot/sync/message.go:94: File is not `gofmt`-ed with `-s` (gofmt)
                if (blockRequest.RequestedData & network.RequestedDataBody)>>1 == 1 {
```

but when I try to make them all without a space I get:
```
dot/sync/message.go:87: File is not `gofmt`-ed with `-s` (gofmt)
                if (blockRequest.RequestedData&network.RequestedDataHeader) == 1 {
```
so this is what gofmt gives us, I don't know why.

Reply:
alright then lol the only diff is this has the bit shift 🤔 very strange, but ok as is then